### PR TITLE
SCSSLintBear.py: Update version of scss_lint

### PR DESCRIPTION
**Pull request by vddesai1998**
Updates version of scss_lint and also modifies tests
according to newer versions of scss_lint.
Now it passes all the tests.

Closes https://github.com/coala/coala-bears/issues/2179

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/scss/SCSSLintBearTest.py [hunk 1]
```diff
@@ -57,6 +57,6 @@
 
 SCSSLintBearChainedClassesTest = verify_local_bear(
     SCSSLintBear,
-    valid_files=(bad_file2, good_file2),
-    invalid_files=(bad_file,),
+    valid_files=(good_file, good_file2),
+    invalid_files=(bad_file,bad_file2),
```
[the remark this answers is not in the document]
Comment (shreyateeza) [reply tests/scss/SCSSLintBearTest.py]: Add space after comma in 
> invalid_files=(bad_file,bad_file2)
then you are good to go!

**Comment by shreyateeza** [comment]
Oh you corrected the mistake already. Good. 
ack a2e22a7

**Comment by shreyateeza** [comment]
ack a2e22a7

File: tests/scss/SCSSLintBearTest.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
 bad_file2 = '''
 $value: 5px;
 
-.foo {
+ .foo {
```
Comment (Kagamihime) [inline tests/scss/SCSSLintBearTest.py]: I think that there is no need to add a space here.

Comment (vddesai1998) [reply tests/scss/SCSSLintBearTest.py]: It's an instance of a bad file. So I added an extra space to test that the linter indeed returns a number message or warning.

Comment (Kagamihime) [reply tests/scss/SCSSLintBearTest.py]: Sorry, my bad

**Comment by Makman2** [comment]
ack a2e22a7276498b3df3f9a250333d124cef16fb7d

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**